src/journal/journal-file.h

```diff
@@ -85,6 +85,7 @@ typedef struct JournalFile {
         bool compress_lz4:1;
         bool seal:1;
         bool defrag_on_close:1;
+        volatile bool archive:1;
```

Comment:
I considered naming this `archive_on_offline` instead, since it would be more consistent with `defrag_on_close`, but then there's `seal` and the others...  *shrug*